src/main/scala/it/bitrock/dvs/api/JsonSupport.scala

```diff
@@ -48,6 +48,7 @@ trait JsonSupport extends SprayJsonSupport with DefaultJsonProtocol {
     override def write(eventPayload: EventPayload): JsValue =
```

Comment:
do you think we should remove this `RootJsonFormat` and create two different format for `TopEventPayload` and `TotalEventPayload`?

Reply:
it would remove the need for a couple of casts.
Instead of completely removing this one, I'd add a `ListEventPayload` to account for the two lists.
too much hierarchy? I wouldn't like to loose the compiler help.

Reply:
I don't think it's a problem to add a `ListEventPayload` but right now we are not using that `RootJsonFormat` for `FlightReceivedList` and `AirportList` so I'm not sure we need a generic `RootJsonFormat[EventPayload]`

Reply:
we are using the writer in the `GlobalMessageDispatcher` actor